contracts/core/AuraMinterOwner.sol

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+
+import { Ownable } from "@openzeppelin/contracts-0.8/access/Ownable.sol";
+import { AuraMinter } from "./AuraMinter.sol";
+
+/**
+ * @title   AuraMinterOwner
+ * @notice  Wraps the AuraMinter mint function and protects from inflation with a cap
+ *          of 3 million AURA per year for 3 years (9 million total cap).
+ * @dev     This contract provides an additional layer of protection over the AuraMinter
+ *          by implementing yearly minting caps and a total cap. Ownership initially
+ *          owned by the DAO. The contract tracks epochs starting from when the
+ *          AuraMinter's inflation protection time expires.
+ * @author  AuraFinance
+ */
+contract AuraMinterOwner is Ownable {
+    /// @notice The AuraMinter contract that this contract wraps
+    /// @dev This contract calls the mint function on the AuraMinter
+    AuraMinter public immutable auraMinter;
+
+    /// @notice Maximum total AURA that can be minted through this contract (9 million with 18 decimals)
+    /// @dev This represents the absolute cap across all epochs (3 years * 3M per year)
+    uint256 public constant MAX_TOTAL_CAP = 9_000_000 * 1e18;
```

Comment:
We don't need to cap the total amount mineable, we just want to cap that only 3M gets minted every 52 weeks.